h2o-r/h2o-DESCRIPTION.template

```diff
@@ -19,7 +19,8 @@ NeedsCompilation: no
 SystemRequirements: Java (>= 7)
 Depends: R (>= 2.13.0),
          methods,
-         stats
+         stats,
+         survival
```

Comment:
can we do suggest only? and check that the survival package is installed when the user wants to use coxph? this could cause issues for some customers with the update

Reply:
The survival package is one of the 15 recommended packages that comes with the pre-built Windows and Mac R installers:

R> installed.packages()["survival", "Priority"]
[1] "recommended"
R> table(installed.packages()[,"Priority"], useNA = "ifany")
base recommended        <NA> 
    14                      15           195 

I guess some folks at large institutions could be forced to use some centralized, locked down Linux installation of just r-base.